.github/workflows/consecutive.yml

```diff
@@ -287,8 +282,6 @@ jobs:
     needs:
       - allure
       - prepare-env
-    env:
-      NETWORK: ${{ needs.prepare-env.outputs.network }}
     steps:
       - name: Send a notification to Teams channel
         uses: neonlabsorg/teams-notification-action@0.1.1
```

Comment:
uses: neonlabsorg/teams-notification-action@1.2
however, prefer avoiding of adding everything in a single PR